<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenWrittersClass</class>
 <widget class="QMainWindow" name="ScreenWrittersClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScreenWritters</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="windowName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="tableLayout">
            <item>
             <widget class="QTableView" name="myTable"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="descriptionLabel">
                <property name="text">
                 <string>Description</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="descriptionBuddy"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="actLabel">
                <property name="text">
                 <string>Act</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="actBuddy"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="addPropButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCommandLinkButton" name="aprooveButton">
          <property name="text">
           <string>Aproove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="savePlotButton">
          <property name="text">
           <string>Save Plot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ScreenWritters.qrc"/>
 </resources>
 <connections/>
</ui>
